src/FOM/UserBundle/Resources/translations/messages.de.xlf

```diff
@@ -594,6 +594,54 @@
         <source>User credentials have expired.</source>
         <target>Anmeldedaten sind abgelaufen.</target>
       </trans-unit>
+      <trans-unit id="fom.user.user.index.datatable.emptytable">
```

Comment:
I just added some translations to existing Translation in FOM. I suggest you open an enhancement issue for that (like change xlf to yaml). 

fom.user.user... yep that's what i found there :)